test/unittests/instantiate_test.cpp

```diff
@@ -18,7 +18,7 @@ uint64_t call_table_func(Instance& instance, size_t idx)
 {
     const auto& elem = (*instance.table)[idx];
     const auto res = elem->function(instance, {}, 0);
-    // ASSERT_TRUE(res.result.has_value());
+    assert(res.has_value);
```

Comment:
Why not keeping `ASSERT_TRUE`? (I'm not sure why it was disabled in this PR in the first.)

Reply:
Apparently the `ASSERT_TRUE` only works inside `TEST` as it use `return` for early exit.

Reply:
Changed to `EXPECT_TRUE`.